Formula/yadm.rb

```diff
@@ -0,0 +1,15 @@
+class Yadm < Formula
+  desc "Yet Another Dotfiles Manager"
+  homepage "https://github.com/TheLocehiliosan/yadm"
+  url "https://github.com/TheLocehiliosan/yadm/archive/1.04.tar.gz"
+  sha256 "a73aa51245866ce67aeb4322a62995ebbb13f29dc35508f486819dceb534968a"
+
+  def install
+    bin.install "yadm"
+    man1.install "yadm.1"
+  end
+
+  test do
+    system "yadm", "version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

You're in a unique `testpath` here that is disposed of after `test do` is executed, so you can play with the directory pretty safely.